pom.xml

```diff
@@ -67,6 +67,7 @@
 				<configuration>
 					<strictBinIncludes>false</strictBinIncludes>
 					<strictVersions>false</strictVersions>
+					<skipPomGeneration>true</skipPomGeneration>
```

Comment:
I'm not sure if this one is needed if the `buildManagement` section below defines it as well. Can you check what the effective POM looks like without this change?

Reply:
You were right, the second one is enough 👍🏻